### PR TITLE
feat:284781 - add countries to cdo

### PR DESCRIPTION
**Pull request by whitewaterdesign**
- contains a security patch - "tough-cookie": ">=4.1.3",
- fixes the 500 instead of 404 on /dog/:index-number and /dog-owner/:index-number
- adds option to add address -> country in POST /cdo

### CODE REVIEW

File: app/routes/dog.js [hunk 1]
```diff
@@ -11,6 +11,10 @@ module.exports = [{
     const indexNumber = request.params.indexNumber
     try {
       const dog = await getDogByIndexNumber(indexNumber)
+
+      if (dog === null) {
+        return h.response().code(404)
+      }
       return h.response({ dog: dogDto(dog) }).code(200)
```
Comment (jbarnsley10) [inline app/routes/dog.js:18]: Very picky but a blank line is nice before the 'return'

File: package.json [hunk 2]
```diff
@@ -38,6 +38,7 @@
     "pg-hstore": "^2.3.4",
     "read-excel-file": "^5.7.1",
     "sequelize": "^6.33.0",
+    "tough-cookie": ">=4.1.3",
```
Comment (jbarnsley10) [inline package.json:41]: Do we need the '>=' or is it better to use '^' ?

**Review by jbarnsley10 — APPROVED** [review]
Some minor comments

File: app/routes/dog.js (hunk 1 above)
Comment (whitewaterdesign) [reply app/routes/dog.js:18]: Good call , have done the same to you

**Pull request merged**